docs/source/conf.py

```diff
@@ -14,12 +14,12 @@
 # -- General configuration ------------------------------------------------
 
 # Build documentation for the following tags and branches
-TAGS = []
-BRANCHES = ['main']
+TAGS = ['v0.9.0']
+BRANCHES = ['main', 'v0.9.0']
 # Set the latest version.
-LATEST_VERSION = 'main'
+LATEST_VERSION = 'v0.9.0'
 # Set which versions are not released yet.
-UNSTABLE_VERSIONS = []
+UNSTABLE_VERSIONS = ['v0.9.0']
```

Comment:
This applies to all drivers, but could this be automated? Meaning this script would automatically add all `vX.Y.Z` tags. From my experience in other drivers (Java), this additional step adds more work while releasing new versions and frankly I forget about it sometimes.

Reply:
@avelanarius We'll look into it. For now, let's try to communicate about the release so that I can take care of publishing the docs.
Adding tags shouldn't be a problem, but also we need to configure which version is the latest and specify deprecated and unstable versions (if any).